### PR TITLE
Stef

### PR DESCRIPTION
**Pull request by stefvanlieshout**
Check pls :)

### CODE REVIEW

File: cram2bam [hunk 1]
```diff
@@ -10,13 +10,10 @@ fi
...
 ## use 80 percent of available threads
-threads=$( expr $(grep -c processor /proc/cpuinfo) / 10 \* 8)
-samtools="/data/tools/samtools_v1.10/samtools"
+#threads=$( expr $(grep -c processor /proc/cpuinfo) / 10 \* 8)
 
-## reset paths for use at datastore server
-if [[ "$(hostname)" == "datastore" ]]; then
-    samtools="/data/common/tools/samtools_v1.10/samtools"
-fi
+threads=3
```
Comment (pauldwolfe) [inline cram2bam]: In general its best to make the image as independent from the machine as possible. So we should either find a way to make the cpu detection magic work or allow the caller pass the threads (fine to default to 3)

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,16 @@ FROM google/cloud-sdk:latest
 
 RUN apt-get update
 RUN apt-get --yes install openjdk-11-jre
+RUN apt-get --yes install picard-tools
```
Comment (pauldwolfe) [inline Dockerfile]: Can combine all these installs into one line `RUN apt-get --yes install openjdk-11-jre picard-tools` and we should specify the versions tool=version.

**Review by pauldwolfe — APPROVED** [review]
Nice work!

File: Dockerfile (hunk 2 above)
Comment (stefvanlieshout) [reply Dockerfile]: I noticed that the downside of using one "RUN apt-get" line, is that when you add one tool, docker needs to rebuild all.

File: cram2bam (hunk 1 above)
Comment (stefvanlieshout) [reply cram2bam]: Agreed. Noticed that pigz defaults to using all available so might not need specific thread count anyway.

Comment (nedleitch) [reply cram2bam]: Also to add some colour to that, in the pipeline we have a few spots where we need to specify a count and we just use a little bit of shell code to determine the number of cores (not dissimilar to the line you commented out). Then we control what that really means by manipulating the sizes of the machines in configuration.

File: Dockerfile (hunk 2 above)
Comment (nedleitch) [reply Dockerfile]: Docker has this concept of "layers" that represent atomic units of change to a base image. If I remember properly each line in the Dockerfile maps to a layer, so this makes sense.

**Review by nedleitch — APPROVED** [review]
Nice work!

**Pull request closed without merging**